testsuite/MDAnalysisTests/coordinates/base.py

```diff
@@ -445,6 +445,33 @@ def test_pickle_reader(self, reader):
         assert_equal(reader.ts, reader_p.ts,
                      "Timestep is changed after pickling")
 
+    
+    @pytest.mark.parametrize('order', ('fac', 'fca', 'afc', 'acf', 'caf', 'cfa'))
+    def test_timeseries_shape(self, reader, order):
+        timeseries = reader.timeseries(order=order)
+        a_index = order.index('a')
+        f_index = order.index('f')
+        c_index = order.index('c')
+        assert(timeseries.shape[a_index] == reader.n_atoms)
+        assert(timeseries.shape[f_index] == len(reader))
+        assert(timeseries.shape[c_index] == 3)
+
+    @pytest.mark.parametrize('slice', ([0,2,1], [0,10,2], [0,10,3]))
+    def test_timeseries_values(self, reader, slice):
+        ts_positions = []
+        if isinstance(reader, mda.coordinates.memory.MemoryReader):
+            pytest.xfail("MemoryReader uses deprecated stop inclusive"
```

Comment:
Raise an issue and tie it to the 3.0 milestone?

Reply:
happy with #3893 to cover it?